docs/arch.md

```diff
@@ -138,12 +144,32 @@ health_statuses_insights{metric="moderate"} 1
 health_statuses_insights{metric="total"} 2
 ```
 
+### Metrics
+
+- `health_statuses_insights`, information about the cluster health status based on the last downloaded report, corresponding to its number of hitting recommendations grouped by severity.
+- `insightsclient_request_send_total`, tracks the number of archives sent.
+- `insightsclient_request_recvreport_total`, tracks the number of insights reports received/downloaded.
+- `insightsclient_last_gather_time`, the time of the last Insights data gathering.
+- `insights_recommendation_active`, expose insights recommendations as Prometheus alerts.
+
+> **Note**
+> The metrics are registered by [the `MustRegisterMetrics` function](../pkg/insights/metrics.go)
+
+### Alerts
+
+- `InsightsDisabled`, insights operator is disabled.
+- `SimpleContentAccessNotAvailable`, simple content access certificates are not available.
+- `InsightsRecommendationActive`, an Insights recommendation is active for this cluster.
```

Comment:
One detail. The word `Insights` with lower case and upper case first letter is mixed here. I think it would be nice to unify it and use only the upper case letter.

Reply:
yeah, good catch... typo